react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+    this.state = {
+      classHid: 'body hidden'
+    };
+  }
+
+  handleClick(e) {
+    const target = e.target.nextElementSibling;
+    target.classList.toggle('hidden');
+
+  }
+
+  render() {
+    const { items } = this.props;
+
+    const arrayItem = items.map(first => [Object.keys(first), Object.values(first)]);
+
+    const elements = arrayItem.map(el => (
+      <div key={el[0]}>
+        <p className='header' onClick={this.handleClick}>{el[0]}</p>
+        <p className='body hidden true'>{el[1]}</p>
+      </div>
+    )
+    );
```

Comment:
Then in your map function use the value in state to determine which classes should be added
```JSX
const elements = items.map(item => (
  <div key={item.id}>
    <p className='header' onClick={() => this.handleClick(item.id)}>{item.title}</p>
    <p className={`body true ${this.state.id === item.id ? '' : 'hidden'}`}>{item.content}</p>
  </div>
));
```